### DISCUSSION TITLE
Postgres' PostGIS GEOMETRY/GEOGRAPHY data types in Keystone?

### DISCUSSION THREAD

**Discussion by arthurCormack** [keystonejs/keystone#9412] [Feature requests]
I have some applications that make use of geospatial data that I am am considering migrating to KeystoneJS. But an essential requirement is POSTGIS' GEOMETRY & GEOGRAPHY data types. Those aren't supported in Prisma, but they can be utilized nonetheless, using Prisma's Raw Queries (https://www.prisma.io/docs/orm/prisma-client/using-raw-sql/safeql#1-set-up-prisma-orm-for-use-with-postgis) . I can imagine a multitude of complications to this, such as the other databases that work with Keystone, not having PostGIS. But it might be possible emulate the same functionality using a collection of other fields in those other databases to achieve the same result, albeit in a less performant way.  
I'm wondering how feasible/likely it might be to create a GEOMETRY/GEOGRAPHY Types in Keystone?
What are the recommendations for handling geospatial data in keystone in the meantime?
Is this something that would be of value to others?

**Comment by arthurCormack** [comment]
It looks like we will have geospatial types support in prisma very soon: https://github.com/prisma/prisma-engines/pull/4208
If and when this is complete, do you think Keystone might consider supporting geospatial types?

**Reply by gautamsi** [reply]
its been pending for long. Once that is released I we probably see some traction here. At least that will open up custom field avenues
